src/config/backpack/settings.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Table Name
+    |--------------------------------------------------------------------------
+    |
+    | Database Settings Table Name
+    |
+    */
+    'table_name' => 'settings',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Route Name
+    |--------------------------------------------------------------------------
+    |
+    | Routing Name
+    |
+    */
+    'route_name' => 'setting',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Config Prefix
+    |--------------------------------------------------------------------------
+    |
+    | The prefix used to add your settings into the configuration array.
+    | With this default you can grab your settings with: config('settings.your_setting_key')
+    |
+    | WARNING: WE ADVISE TO NOT LEAVE THIS EMPTY / CHECK IF IT DOES NOT CONFLICT WITH OTHER CONFIG FILE NAMES
+    |
+    |   - if you leave this empty and your keys match other configuration files you might ovewrite them.
+    |
+    */
+    'config_prefix' => 'settings',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Migration file name
+    |--------------------------------------------------------------------------
+    |
+    | The file name for the settings migration file. It will be created in database_path('/migrations/%name_you_choose%.php)
+    | Note: .php extension is automatically added.
+    |
+    */
+    'migration_name' => 'create_settings_table',
```

Comment:
```suggestion
    'migration_name' => '2015_08_04_131614_create_settings_table',
```

@pxpm should we keep the current name(date)? And also induce devs to keep the Laravel standard of having the date in the Laravel migrations?


@pxpm sorry for making so much questions 😅

Reply:
Other than this, everything is ok 👌

Reply:
This is because of this;
```php
$this->publishes([
    __DIR__.'/database/migrations/create_settings_table.php.stub' => database_path('migrations/'.config('backpack.settings.migration_name').'.php'),
], 'migrations');
```

Migration name will be only `create_settings_table` right?
![image](https://user-images.githubusercontent.com/1838187/134041741-02e5a5d5-9329-43f3-9122-d59915726e4e.png)

Reply:
Totally, without this it would be a BC. 

Nice catch! Thanks @promatik